assets/scripts/script.js

```diff
@@ -1,10 +1,64 @@
 $(document).ready(function () {
 
+    var arr = JSON.parse(localStorage.getItem("city")) || [];
+    var queryURL = " https://api.covidtracking.com/v1/us/current.json";
+    $.ajax({
+        type: "GET",
+        url: queryURL,
+        dataType: "JSON",
+        success: function (response) {
+            console.log(response);
+        }
+    });
+
+    if (arr.length > 0) { 
+        arr.forEach(name => { 
+            createCitylist(name);
+        })
+    }
+    
+
     $("#search-form").on("submit", function (event) {
         event.preventDefault();
         var searchInput = $("#city-text").val().trim();
 
         console.log(searchInput);
+        if (arr.indexOf(searchInput) === -1) { 
+            createCitylist(searchInput);
+            arr.push(searchInput);
+            localStorage.setItem("city", JSON.stringify(arr));
+            // getCurrentConditions();
+            }
+    })
+
+    function createCitylist(name) { 
+        var id = 'btn' + name
+        var li = $("<li >");
+        var btn = $('<button />', {
+            text: 'remove',
+            id: id,
+            click: function () {
+                $(this).parent().hide();
+                var localData = JSON.parse(localStorage.getItem("city"));
+                if (localData) {
+                    var data = localData.filter(city => city !== name);
+                    localStorage.removeItem('city');
+                    localStorage.setItem('city', JSON.stringify(data));
+                }
+            }
+        });
+        li.addClass("city-list-item");
+        li.text(name).append(btn);
+        $("#city-list").append(li);
+
+        li.on("click", function (event) {
+            event.preventDefault();
+            name = $(this).text();
+            console.log(name);         
+        });
+    }
+
+    function getCurrentConditions(response) {
```

Comment:
Line 65 has an extra ")" that is causing an error. Once you remove that, everything should be good to go.